pyqir/src/builder.rs

```diff
@@ -199,6 +201,21 @@ impl Builder {
         }
     }
 
+    /// Converts a dynamically-calculated qubit index to a Qubit pointer
+    ///
+    /// :param Value id: The qubit id as a Value
+    /// :returns: A Value for a Qubit pointer
+    /// :rtype: Value
+    #[pyo3(text_signature = "(self, id)")]
+    fn dyn_qubit(&self, py: Python, id: &Value) -> PyResult<PyObject> {
```

Comment:
Not the most beautiful name or place for this, but I needed it and couldn't find it anywhere. All I wanted was to create an [`inttoptr`][1] with an input SSA edge but `pyqir.qubit()` seems to expect a constant `int`

[1]: https://llvm.org/docs/LangRef.html#inttoptr-to-instruction